application/main.py

```diff
@@ -70,13 +76,21 @@ def log_individual_service_responses(self):
     def send_request(self, request: bytes):
         endpoint = os.environ.get("DRD_ENDPOINT")
         basic_auth = os.environ.get("DRD_BASICAUTH")
-        r = requests.post(url=endpoint, data=request, headers={
-                "Authorization": basic_auth,
-                "Content-type": "application/octet-stream",
-                "Accept": "application/octet-stream"
-            }
-        )
-        self.status_code = r.status_code
+        mock_mode = os.environ.get("DRD_MOCK_MODE")
+        if (mock_mode == "False"):
```

Comment:
Only question here is that I was under the impression we would have a separate endpoint for mock functions, rather than weaving test code into production code?

Reply:
The only issue with that is it needs different information sent through - let's discuss it, though